beacon_localization:
    scan_topic: /scan
    post_distance: 1.372    # Distance posts are apart from one another on beacon 
    post_distance_err: 0.15
    left_post_loc: [-0.686, 0]
    right_post_loc: [0.686, 0]
    post_width: 0.223
    post_width_err: 0.4
